<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categoriesPage</class>
 <widget class="QWidget" name="categoriesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>462</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="KPushButton" name="manageButton">
             <property name="text">
              <string>Manage</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Categories</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KListWidget" name="enabled_categories_listwidget"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <layout class="QHBoxLayout" name="horizontalLayout_4"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="KTabWidget" name="category_tabwidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="plugins_tab">
          <attribute name="title">
           <string>Category plugins</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="KListWidgetSearchLine" name="klistwidgetsearchline"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Category settings</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>299</width>
                <height>395</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Query</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="KPushButton" name="query_show_button">
                   <property name="text">
                    <string>show&gt;&gt;</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="0" colspan="3">
                  <widget class="QWidget" name="query_edit_widget" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>10</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_3">
                    <item row="0" column="1">
                     <widget class="QCheckBox" name="rawQueryEditCheckBox">
                      <property name="text">
                       <string>Raw editing</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="2">
                     <widget class="QStackedWidget" name="queryEditStackedWidget">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <widget class="QWidget" name="page">
                       <layout class="QVBoxLayout" name="verticalLayout_4">
                        <item>
                         <widget class="Nepomuk::Utils::SearchWidget" name="queryBuilder" native="true"/>
                        </item>
                       </layout>
                      </widget>
                      <widget class="QWidget" name="page_3">
                       <layout class="QVBoxLayout" name="verticalLayout_3">
                        <item>
                         <widget class="KTextEdit" name="query_edit">
                          <property name="documentTitle">
                           <string/>
                          </property>
                          <property name="acceptRichText">
                           <bool>false</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Query prefix</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="KPushButton" name="prefix_show_button">
                   <property name="text">
                    <string>show&gt;&gt;</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="0" colspan="3">
                  <widget class="KTextEdit" name="query_prefix_edit">
                   <property name="documentTitle">
                    <string/>
                   </property>
                   <property name="acceptRichText">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Polling interval</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="KIntSpinBox" name="interval_spinbox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>Plugin select algorithm</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_15">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="KComboBox" name="plugin_algo_combobox">
                   <property name="autoCompletion">
                    <bool>false</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>All</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Stepwise</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="managePage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="KPushButton" name="returnButton">
         <property name="text">
          <string>Return</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="KActionSelector" name="category_selector">
         <property name="showUpDownButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KPushButton" name="add_button">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="KPushButton" name="remove_button">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KActionSelector</class>
   <extends>QWidget</extends>
   <header>kactionselector.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>klistwidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>Nepomuk::Utils::SearchWidget</class>
   <extends>QWidget</extends>
   <header location="global">nepomuk/searchwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>query_show_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>query_edit_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prefix_show_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>query_prefix_edit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
